package.json

```diff
@@ -11,11 +11,15 @@
 		"url": "sindresorhus.com"
 	},
 	"engines": {
-		"node": ">=8"
+		"node": ">=10"
```

Comment:
Node.js 12 can be targeted.

Reply:
Done, also removed node 10 from github actions